integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/datamap/CarbonDataMapShowCommand.scala

```diff
@@ -43,7 +44,8 @@ case class CarbonDataMapShowCommand(tableIdentifier: Option[TableIdentifier])
     Seq(AttributeReference("DataMapName", StringType, nullable = false)(),
       AttributeReference("ClassName", StringType, nullable = false)(),
       AttributeReference("Associated Table", StringType, nullable = false)(),
-      AttributeReference("DataMap Properties", StringType, nullable = false)())
+      AttributeReference("DataMap Properties", StringType, nullable = false)(),
+      AttributeReference("Subquery", StringType, nullable = false)())
```

Comment:
How about setting this field nullable so that in case of other type of datamap we can just return null instead of empty.